### ISSUE TITLE
mouseWheelBehavior for XYChart

### ISSUE THREAD

**Issue by enriquepiatti** [amcharts/amcharts4#1549]
I can see there is a property "mouseWheelBehavior" in MapChart:   
https://www.amcharts.com/docs/v4/reference/mapchart/   
Is not possible to use the same for XYChart?

**Comment by zeroin** [comment]
It is: https://www.amcharts.com/docs/v4/reference/xychart/#mouseWheelBehavior_property

**Comment by enriquepiatti** [comment]
Perfect, dumb question. I hope this is a better one:   
Is possible to control the wheel sensitivity?

**Comment by martynasma** [comment]
On `MapChart` each turn of of the wheel produces one step zoom in/out.

Which is controlled by map's `zoomStep` property.

https://www.amcharts.com/docs/v4/reference/mapchart/#zoomStep_property

I understand that changing it might not be ideal because it also affects other zooming behavior such as ZoomControl.

Introducing some wheel-related sensitivity modifiers would be a good idea. I posted a new issue for that: #1553 (make sure you follow it)

**Comment by enriquepiatti** [comment]
I'm working with XYChart, is there some zoomStep for it? or only for MapChart?

**Comment by martynasma** [comment]
No. For the rest of them there's new sensitivity mouse setting, in just released 4.5.14.

```
chart.plotContainer.interactions.mouseOptions.sensitivity = 0.5;
```

**Comment by enriquepiatti** [comment]
Perfect, great work!
